core/ts/Poll.ts

```diff
@@ -181,33 +176,22 @@ export class Poll implements IPoll {
    * @param index - The index of the message to process.
    * @returns A number of variables which will be used in the zk-SNARK circuit.
    */
-  private processMessage = (index: number): IProcessMessagesOutput => {
-    //TODO: throw custom errors for no-ops
-
+  public processMessage = (message: Message, encPubKey: PubKey): IProcessMessagesOutput => {
```

Comment:
Perhaps doesn't matter but why change this to a public method?

Reply:
So we can unit test it

Reply:
Gotcha. Is that the same reason you're now passing `encPubKey` into the function?

Reply:
Not only, we already extracted the message in the line before, so thought we might as well pass the message we already extracted (to check the msgType), so I added the key as param too:

https://github.com/privacy-scaling-explorations/maci/blob/dev/core/ts/Poll.ts#L451-L455